stylelint-plugin/rules/use-proper-token/index.test.ts

```diff
@@ -0,0 +1,103 @@
+import { describe, it, expect } from 'vitest'
+import getInappropriateTokens from './getInappropriateTokens'
+
+describe('getInappropriateTokens', () => {
+  it('returns empty array when no token used in value', () => {
+    const decl = {
+      prop: 'color',
+      value: 'red',
+    }
+
+    expect(getInappropriateTokens(decl)).toEqual([])
+  })
+
+  it('returns empty array when no `kui-` token used in value (CSS variable)', () => {
+    const decl = {
+      prop: 'color',
+      value: 'var(--color-red)',
+    }
+
+    expect(getInappropriateTokens(decl)).toEqual([])
+  })
+
+  it('returns empty array when no `kui-` token used in value (SCSS variable)', () => {
+    const decl = {
+      prop: 'color',
+      value: '$color-red',
+    }
+
+    expect(getInappropriateTokens(decl)).toEqual([])
+  })
+
+  it('returns empty array when property is not enforced', () => {
+    const decl = {
+      prop: 'border-collapse',
+      value: 'var(--kui-color-text)',
+    }
+
+    expect(getInappropriateTokens(decl)).toEqual([])
+  })
```

Comment:
I'm not sure I understand what "is not enforced" means here?

Reply:
Rephrased